### ISSUE TITLE
Support for private repos?

### ISSUE THREAD

**Issue by danmakenoise** [juliangruber/travis-watch#28]
This looks like a great tool we could use in my team, but it doesn't seem to work with private repositories. Happy to help code a solution, but wanted to put an issue in to see if that is something you would like to support or not.

**Comment by juliangruber** [comment]
Yes I'm definitely interested in supporting private repos :) Hopefully this will just mean adding some means of authentication and talking to a different api endpoint.

**Comment by juliangruber** [comment]
Glad to receive help here :)

**Comment by JuanCaicedo** [comment]
I'm very interested in having this for private repos, I can offer some help 😃 Can you point to somewhere in the code I should look at?

**Comment by juliangruber** [comment]
That's great! I haven't checked out travis' private repo api yet, but we're instantiating the travis api module [here](https://github.com/juliangruber/travis-watch/blob/master/index.js#L11) and then are using it through the 163L of logic in https://github.com/juliangruber/travis-watch/blob/master/index.js
